keras_nlp/models/albert/albert_masked_lm.py

```diff
@@ -142,6 +143,13 @@ def __init__(self, backbone, preprocessor=None, **kwargs):
         self.backbone = backbone
         self.preprocessor = preprocessor
 
+        self.compile(
+            loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+            optimizer=keras.optimizers.Adam(5e-5),
+            metrics=keras.metrics.SparseCategoricalAccuracy(),
```

Comment:
Actually, I think we want weighted_metrics here as we do pass sampler weights for this task.

Reply:
Sure Matt.